app/dict_to_property/dict_to_armour.py

```diff
@@ -0,0 +1,10 @@
+from app.knight_properties.armour import Armour
+
+
+def dict_to_armour(value: dict) -> list[Armour]:
+    armour = []
+    if len(value["armour"]):
+        for armour_item in value["armour"]:
+            armour.append(Armour(armour_item["part"],
+                                 armour_item["protection"]))

```

Comment:
use list comprehension here

Reply:
also I think no need to create `app/dict_to_property/dict_to_armour.py`
you can store `dict_to_armour ` in `app/knight_properties/armour.py`, same with `app/dict_to_property/dict_to_potion.py`

Reply:
Am I should put dict_to_armour inside class Armouar?

Reply:
Done.